### DISCUSSION TITLE
Possibility to set font size in config

### DISCUSSION THREAD

**Discussion by Avaray** [saadeghi/daisyui#2826] [Ideas / feature requests]
I know it is possible to set font size [with CSS](https://daisyui.com/docs/themes/#-6).

Now imagine I have 20 themes with 20 different custom fonts and I need to set font size for each of them in my CSS file.

I think it's better to have less code :)
I see it as `fontSize` key in theme object.

**Comment by NikarashiHatsu** [comment]
The font's unique characteristics, with varying heights and widths for each character, often lead to edge cases. However, there's little necessity to delve into such intricacies to determine whether a developer opts for Font A, Font B, or even Font n. Assessing the problem at hand, incorporating custom fonts within each theme proves less than ideal in web development scenarios.

Honestly, the onus lies on you to meticulously tweak the `fontSize` for each font manually, rather than relying on the library to handle this task seamlessly.
